### PR TITLE
Add `parent` to Blocks

### PR DESCRIPTION
**Pull request by Fosome**

### CODE REVIEW

**Comment by Fosome** [comment]
Guess I'll need to fix the keyword args here to support ruby 1.9.x

**Comment by brianjlandau** [comment]
Since it's mostly an internal API, yeah I'd say keep it 1.9 compatible. If the developer facing API used keywords and that made it substantially more expressive I'd say drop 1.9 support.

:+1: Also, I like this feature.

File: lib/colonel_kurtz/version.rb [hunk 1]
```diff
@@ -3,6 +3,6 @@ module ColonelKurtz
   VERSION = [
     0,    # major
     0,    # minor
-    2     # patch
+    3     # patch
```
Comment (brianjlandau) [inline lib/colonel_kurtz/version.rb]: Seems like this is a minor bump, not patch bump. It's not a bug fix, it's a change to how that object works. [semver](http://semver.org/)

Comment (Fosome) [inline lib/colonel_kurtz/version.rb]: I agree.  Updating.

**Comment by ltk** [comment]
:+1:

**Comment by dce** [comment]
:+1:

**Pull request closed without merging**